proto/regen/ecocredit/v1/tx.proto

```diff
@@ -84,6 +102,13 @@ message MsgCreateClass {
   // acceptable fees for creating a credit class can be found in the governance
   // parameters for the ecocredit module.
   cosmos.base.v1beta1.Coin fee = 5;
+
+  // allow_pending_issuances enables permissionless creation of pending
+  // batches & batch issuances. For use in credit classes that have
+  // bridged assets, and also credit classes that want to allow for
+  // users to propose credit issaunces for a specific project (e.g.
+  // from the project developer).
+  boolean allow_pending_issuances = 6;
```

Comment:
how do we plan to update this field value for existing credit classes (eg C01) as part of future upgrades if we want it to be true?
do we want to do it manually or add some msg for updating this?
Allowing for updates of such a field might get tricky.

Reply:
I think making it updatable actually might make sense.  @S4mmyb what do you think? Is there a use case where someone may instantiate a credit class, and at a later date decide that they want to allow pending issuances?

I think from a technical perspective it shouldn't be too tricky:
1.  If a credit class starts out not allowing pending issuances, then allowing pending issuances later shouldn't require any special logic
2. if a credit class starts out allowing pending issuances, then disallowing pending issuances later should just immediately disapprove any pending issuances
I can't think of a use case for a user doing (2) in practice, but I also don't see why the technical solution described in (2) would be bad. If an admin wants to turn off pending issuance support, then it seems reasonable to wipe out any currently pending batches/issuances.

Reply:
yeah my main question was about (2), whether we want to disallow existing pending issuances or only future ones. But seems like disallowing all seems more accurate indeed.

Reply:
I agree with @blushi on this one. I can't really think of too many use cases for this to be honest, but I agree including that functionality isn't a bad idea.